File: src/main/java/utils/JsonUtils.java
package utils;

import org.json.JSONArray;
import org.json.JSONException;
import org.json.JSONObject;

public class JsonUtils {
    
    static public JSONObject stringToJsonObject(String s) {
        return new JSONObject(s);
    }
    
    static public JSONArray stringToJsonArray(String s) {
	    return new JSONArray(s);
    }

    static public String getStringOrNull(JSONObject jsonObject, String key) {
        try {
            return jsonObject.getString(key);
        } catch (JSONException e) {
            return "";
        }
    }

}

File: src/main/kotlin/utils/JsonUtils.kt
package utils

import org.json.JSONArray
import org.json.JSONException
import org.json.JSONObject

object JsonUtils {

    fun stringToJsonObject(s: String): JSONObject {
        return JSONObject(s)
    }

    fun stringToJsonArray(s: String): JSONArray {
        return JSONArray(s)
    }

    fun getStringOrNull(jsonObject: JSONObject, key: String): String {
        try {
            return jsonObject.getString(key)
        } catch (e: JSONException) {
            return ""
        }

    }

}
